packages/backend/src/apps/formsg/__tests__/triggers/new-submission.test.ts

```diff
@@ -16,6 +16,7 @@ describe('new submission trigger', () => {
             answer: 'herp derp',
             fieldType: 'textField',
             order: 1,
+            answerArray: ['answer 1', 'answer 2'],
```

Comment:
nit: `answer` and `answerArray` are actually [mutually exclusive](https://github.com/opengovsg/formsg-javascript-sdk/blob/7ba2a384f81805654a22e21cf1f96b55b7bc195a/src/types.ts#L38), it would be better to have this as separate test case.